megamek/src/megamek/client/ui/swing/CommonSettingsDialog.java

```diff
@@ -605,7 +594,7 @@ private JPanel getSettingsPanel() {
         row = new ArrayList<>();
         row.add(getFocus);
         comps.add(row);
-
+        
```

Comment:
```suggestion

```

Reply:
This empty line separates the option blocks. Not different from many before and after. Dont think this should be changed.

Reply:
This removes the extra spacing from it, which would be automatically removed in a further commit. Just dealing with future blankspace changes now.